buildbot/sycl-bisect.bash

```diff
@@ -0,0 +1,195 @@
+#!/bin/bash
+
+# This is a script for running git-bisect on sycl, sycl-web, and sycl pulldown
```

Comment:
These docs are good, but could you include an example of running it?
When bisecting "normally" I usually just do `git bisect start` `git bisect good xxxx` `git bisect bad xxxx` `git bisect run my_test_script.sh` . But its unclear to me exactly how to use this one, maybe because I'm not appreciating the context of its use fully, having not worked with the llvm pulldown difficulties very much.

Reply:
I've added a usage example to the help text. I'm not sure if you're aware of a similar tool we have for our internal compiler repository called "Xmain-aware Git Bisect", but the way this tool is run is inspired by that. It's very similar to the normal git-bisect procedure, except that you specify the bad commit, good commit(s), and what to test all in one step instead of in multiple stages.

You can also use test-commit-sycl-bisect.bash standalone with the normal git-bisect flow if you use the environment variables as described at the top of that file, but this is a more advanced usage and you miss out on some of the up-front setup/validation that sycl-bisect.bash does. Also, if test-commit-sycl-bisect.bash changes/doesn't exist somewhere in your bisection range this can cause issues (which sycl-bisect.bash works around by copying test-commit-sycl-bisect.bash to a temporary file before starting the bisection).